yt_dlp/extractor/generic.py

```diff
@@ -145,12 +150,12 @@ class GenericIE(InfoExtractor):
             'playlist': [{
                 'info_dict': {
                     'ext': 'm4a',
-                    'id': 'c1c879525ce2cb640b344507e682c36d',
+                    'id': '818a5d38-01cd-152f-2231-ee479677fa82',
```

Comment:
Investigation is needed as to why id changed.

Reply:
Seems like a format change. I checked the video, though, it matches what is played on the web page.

Reply:
imo this should be reverted for now

Reply:
I think it's fine. This is the id of the episode now. This is what's in the RSS XML now